src/openknotscore/substation/README.md

```diff
@@ -0,0 +1,4 @@
+# Substation
```

Comment:
Substation could use more detail, at least a high-level overview of how it is intended to work and what the subsystems are for.